tests/legacy-cli/e2e/tests/i18n/ivy-localize.ts

```diff
@@ -0,0 +1,151 @@
+import * as express from 'express';
+import { resolve } from 'path';
+import { getGlobalVariable } from '../../utils/env';
+import {
+  appendToFile,
+  copyFile,
+  expectFileToExist,
+  expectFileToMatch,
+  replaceInFile,
+  writeFile,
+} from '../../utils/fs';
+import { ng, npm } from '../../utils/process';
+import { updateJsonFile } from '../../utils/project';
+import { expectToFail } from '../../utils/utils';
+import { readNgVersion } from '../../utils/version';
+
+export default async function() {
+  if (getGlobalVariable('argv').ve) {
+    return;
+  }
+
+  let localizeVersion = '@angular/localize@' + readNgVersion();
+  if (getGlobalVariable('argv')['ng-snapshots']) {
+    localizeVersion = require('../../ng-snapshot/package.json').dependencies['@angular/localize'];
```

Comment:
`@angular/localize` needs to be added to `ng-snapshot/package.json` currently it is missing

Reply:
It was.  The rebase appears to have removed it.  Will fix.